src/main/scala/org/labrad/registry/Registry.scala

```diff
@@ -157,9 +157,9 @@ extends LocalServer with Logging {
       newDir
     }
 
-    @Setting(id=10,
-             name="cd",
-             doc="Change the current directory")
+    @Setting(id = 10,
+             name = "cd",
+             doc = """Change the current directory.""")
```

Comment:
Optionally takes a second boolean argument to create the directory if it does not exist.

Also, you should document the integer argument, including stating that it is deprecated if you want to do that.